eng/referenceAssemblies.props

```diff
@@ -7,10 +7,11 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(IsReferenceAssembly)' == 'true'">
+    <AdditionalBuildTargetFrameworks>$(AdditionalBuildTargetFrameworks);netstandard2.0;netstandard2.1</AdditionalBuildTargetFrameworks>
```

Comment:
Does this need to be here rather than ref.builds?  This will make it so that we always build for many target frameworks (eg from commandline, or SLN).  Before that only happened for the repo build.